game/script-jokes.rpy

```diff
@@ -631,3 +659,74 @@ label joke_con_crete:
     extend 1nchgn "-crete!"
 
     return
+
+label joke_footless_snakes:
+    n 1ulraj "Hey,{w=0.2} [player]...{w=1}{nw}"
+    extend 1tnmfl " have you ever wondered how to measure snakes?"
+    n 1uwdaj "Like they are so long!"
+    n 1fsqcs "What measurement could be possibly the best?"
```

Comment:
```suggestion
    n 1uwdaj "Especially with all those different sizes!"
    extend 1fsqcs " What kind of measurement would even work {i}best{/i}?"
```

I get where you're coming from here! But it's also important to remember there's some extremely small snakes too - and noting the differences in sizes instead of just length would work better for the punchline mentioning different units!